tests/test_choose_disk.py

```diff
@@ -0,0 +1,159 @@
+import imp
+import inspect
+import mock
+import os
+from nose.tools import assert_equals, nottest
```

Comment:
`nose` is a test runner/framework that is no longer maintained. I would suggest using `py.test` here which is what we use on most of our projects